IdentityCore/src/telemetry/MSIDTelemetryAPIEvent.m

```diff
@@ -140,3 +142,58 @@ - (void)setOauthErrorCode:(NSString *)oauthErrorCode
 }
 
 @end
+
+#else // MSAL CPP
+
+#import "MSIDTelemetryAPIEvent.h"
+
+@implementation MSIDTelemetryAPIEvent
+
+- (void)setCorrelationId:(NSUUID *)correlationId
```

Comment:
Why those empty APIs are needed?

Reply:
Hi Olga, I double checked and did not find a way to avoid implementing these empty functions. The reference of these functions locates at different places that are not excluded by MSAL CPP, like:
https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/2aa88400f0257c5442f46dcdf4ac3a4708434e48/IdentityCore/src/controllers/MSIDBaseRequestController.m#L80
https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/2aa88400f0257c5442f46dcdf4ac3a4708434e48/IdentityCore/src/controllers/MSIDLocalInteractiveController.m#L169
Without these definitions MSAL CPP builds will fail.

Reply:
I don't think this will be a scalable solution to have duplicate empty functions for all telemetry APIs

Reply:
I believe it's resolved, please take a look, thanks!